com.gitlab.JakobDev.jdTextEdit.yml

```diff
@@ -0,0 +1,30 @@
+app-id: com.gitlab.JakobDev.jdTextEdit
+runtime: org.kde.Platform
+runtime-version: "6.2"
+sdk: org.kde.Sdk
+command: jdTextEdit
+finish-args:
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --share=network
+  - --socket=cups
+  - --device=dri
+  - --share=ipc
+
+modules:
+  - pyqt6.yml
+  - qscintilla-pyqt6.yml
```

Comment:
why splitting these? It's harder to maintain. (including enchant)

Reply:
I personally find it easier to maintain

Reply:
Please merge them back into single file. The usual convention is to maintain everything in a single manifest, unless a specific module is very complicated to build or has many dependencies.